realm/realm-library/src/main/java/io/realm/internal/Group.java

```diff
@@ -226,6 +226,15 @@ public void writeToFile(File file, byte[] key) throws IOException {
         return nativeWriteToMem(nativePtr);
     }
 
+    /*
+     * Check if the Group contains any objects.
+     *
+     * @return {@code true} if empty, @{code false} otherwise.
+     */
+    public boolean isEmptyTables() {
```

Comment:
I think there is a distinction between an empty group (no tables) and a group with tables but no objects. I would keep `Group.isEmpty()` and `Group.size()` for now as we might need them for other things.

I am not a big fan of this method name as it seems a bit ambiguous to me. Maybe `isRealmObjectTablesEmpty` instead?